src/internal/minikubetestenv/deploy.go

```diff
@@ -69,6 +69,15 @@ type DeployOpts struct {
 
 type helmPutE func(t terraTest.TestingT, options *helm.Options, chart string, releaseName string) error
 
+func getLocalImage() string {
+	sha := os.Getenv("TEST_IMAGE_SHA")
```

Comment:
Best style here is to keep the variable binding in the if statement, and to not indent the "default":

```go
if sha := os.Getenv("TEST_IMAGE_SHA"); sha != "" {
    return sha
}
return localImage
```